script/util/presets.js

```diff
@@ -70,6 +72,77 @@ export async function presets(base) {
         ] = option
       }
 
+      if (typeof settings.fences === 'boolean') {
```

Comment:
I think this is the incorrect way to go about this.
The preset readmes lists rules with options.
You removed some options and added shared settings.
Instead of documenting that rules are given options (which isn’t true), wouldn’t it be easier (and more true) to document which settings are defined by the preset?